crates/common/tedge_config/src/settings.rs

```diff
@@ -300,3 +300,27 @@ impl ConfigSetting for TmpPathDefaultSetting {
 
     type Value = FilePath;
 }
+
+pub struct LogPathDefaultSetting;
+
+impl ConfigSetting for LogPathDefaultSetting {
+    const KEY: &'static str = "logs.path";
```

Comment:
@didier-wenzek, @itsyitsy  let me know if this is the correct key for logs.path

Reply:
Yes, I'm fine with `logs.path`.